assets/src/containers/AssignmentPlanning.js

```diff
@@ -40,13 +40,15 @@ const styles = theme => ({
   }
 })
 
-const getCurrentWeek = assignmentData => {
-  assignmentData.forEach(item => {
+export const getCurrentWeek = assignmentData => {
+  let currentWeek = null
+  assignmentData.some((item) => {
     let weekStatus = item.due_date_items[0].assignment_items[0].current_week
     if (weekStatus) {
-      return item.week
+      return currentWeek = item.week
```

Comment:
Does this assignment do anything?

Reply:
yes, it returns current week if there is one and since it is going over the loop for finding the current it just exits the loop once found and `ForEach` was not doing it.